include/misc/stack.h

```diff
@@ -79,7 +79,9 @@ static inline void stack_analyze(const char *name, const char *stack,
 {
 }
 #endif
-
+/**
+ * @brief Analyse stacks
```

Comment:
Analyze 

Also, not a very enlightening comment about this define.

Reply:
I added those to fix reference issues, not that I was planning originally to fix them, the build fails with those missing, if you have a better suggestion, please let me know. :)